src/Component.ts

```diff
@@ -3857,6 +3857,11 @@ class Each extends AbstractElementMediator<any[], HTMLElement, Params> {
 	}
 
 	public wire(): void {
+		if (!this.getEl().querySelector("template[c:type=item]")) {
```

Comment:
This won't work either, as it doesn't address the potential of template not being a direct child of the element and also, the prefix can be something other than c.